speech-commands/src/generic_utlis_test.ts

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
Filename maybe has a typo? Should it be "generic_utils_test.ts" rather than "generic_utlis_test.ts"

Reply:
Fixed. Thanks for catching that!